dom/src/main/scala/fs2/dom/Navigator.scala

```diff
@@ -27,6 +28,10 @@ abstract class Navigator[F[_]] private {
 
   def onLine: F[Boolean]
 
+  def languages: F[js.Array[String]]
```

Comment:
The goal of fs2-dom is to wrap (mutable, unsafe) JS APIs with (immutable, safe) Scala / Cats Effect APIs.
```suggestion
  def languages: F[List[String]]
```

Reply:
Resolved it